CHANGES.rst

```diff
@@ -4,7 +4,9 @@ z3c.recipe.usercrontab changes
 1.4 (unreleased)
 ----------------
 
-- Nothing changed yet.
+- Add enabled-option. By setting this to False, you can generate
```

Comment:
AddED. We're not giving the reader of the changelog the order to add an option. (Read: I totally utterly detest git's "write it as an order to the version control system" commit message style that makes no grammatical sense :-) )

"Add `enabled` option."